.eslintrc

```diff
@@ -3,11 +3,10 @@
     "eslint:recommended"
   ],
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 2018,
```

Comment:
I'm not sure if you should use the latest version, since node is not on par with all the features.

Reply:
I had to make this change to avoid deprecation warnings on the latest eslint 5 version. From [the docs](https://eslint.org/docs/user-guide/migrating-to-5.0.0#-the-experimentalobjectrestspread-option-has-been-deprecated):

> For compatibility, ESLint v5 will treat ecmaFeatures: { experimentalObjectRestSpread: true } as an alias for ecmaVersion: 2018 when the former is found in a config file.
...
If you use the experimentalObjectRestSpread option, you should be able to upgrade to ESLint v5 without any changes, but you will encounter a deprecation warning. To avoid the warning, use ecmaVersion: 2018 in your config file rather than ecmaFeatures: { experimentalObjectRestSpread: true }